drv/Iterators.drv

```diff
@@ -828,19 +828,29 @@ public final class ITERATORS {
 		return new ListIteratorWrapper KEY_GENERIC_DIAMOND(i);
 	}
 	
+	/**
+	 * Returned whether any element satisfies the given predicate.
+	 * Short circuit evaluation is performed; the first {@code true} from the predicate terminates the loop.
+	 */
```

Comment:
One observation: for all methods that require an iterator and do not use the additional methods of Iterator.drv, the right macro is STD_KEY_ITERATOR, which expands to Iterator for objects. That makes all the methods more generic at no cost. I would replace everywhere.

Reply:
And I believe there is at least one method in Spliterators.drv that might benefit from that approach.

Reply:
Both are done